src/matchmaking/tournament/tournament_manager.cpp

```diff
@@ -77,6 +77,7 @@ void TournamentManager::validateEngines() const {
     }
 
     for (std::size_t i = 0; i < engine_configs_.size(); i++) {
+        if (engine_configs_[i].name == "") {throw std::runtime_error("Error; please specify a name for each engine!");}
```

Comment:
sure is correct, the more c++ idomatic way would be
```cpp
if (engine_configs_[i].name.empty())
```

Reply:
fixed 👍